modules/ROOT/pages/_partials/api-mgr.adoc

```diff
@@ -93,7 +93,7 @@ Besides the default `--help`, general, and authentication flags, this command al
 |===
 |Flag |Description 
 |`--limit <value>` | Number of results to retrieve, default is 10 results
-| `--offset <value>` | Offsets the amount of APIs passed, for example`--offset 3`
+| `--offset <value>` | Offsets the amount of APIs passed, for example, `--offset 3`
 | `--output <value>`| Specifies the response format. + 
 Supported values are `table` (default) and `json`
```

Comment:
I'm honestly not sure if this should be considered a complete sentence without it being "The supported values."

I would ask the editors.